### PR TITLE
Introduce XactEng database v2

### PR DESCRIPTION
**Pull request by jarupxx**
Source:
Lord of the Rings - The Two Towers (4928)
Super Bubble Pop (5120)
Cyber Attack Team (5233)
Syberia (5344)
Top Spin (5558)
Steel Battalion - Line of Contact (5788)
Souece: Forza Motorsport (5849)

### CODE REVIEW

File: src/CxbxKrnl/HLEDataBase.cpp [hunk 1]
```diff
@@ -214,7 +220,7 @@ const HLEDataV2 HLEDataBaseV2[] = {
     // I have not seen which header section it supposed to go with...
     //{ Lib_XACTENG, { Sec_???? }, XACTENG_OOVPAV2, XACTENG_OOVPA_SIZEV2 },
```
Comment (RadWolfie) [inline src/CxbxKrnl/HLEDataBase.cpp]: Please uncomment this and replace Sec_???? to Sec_XACTENG.

Then start checking your OOVPAs to be sure they're detecting to correct address and are in correct last early known XDK revision number.

File: src/CxbxKrnl/HLEDataBase.cpp [hunk 2]
```diff
@@ -214,7 +220,7 @@ const HLEDataV2 HLEDataBaseV2[] = {
...
 
-    //
+    // test case: Power Drome (Unluckily, use LTCG.)
     //{ Lib_XACTENLT,{ Sec_XACTENG }, XACTENG_OOVPAV2, XACTENG_OOVPA_SIZEV2 },
```
Comment (RadWolfie) [inline src/CxbxKrnl/HLEDataBase.cpp]: Are you saying XACTENLT library is a LTCG? Or just the Power Drome title is a LTCG for D3D8?

Comment (PatrickvL) [reply src/CxbxKrnl/HLEDataBase.cpp]: @jarupxx Please create a game-compatibility issue for that title, and attach an Xbe dump to it. That way, we can verify the Xbe sections, and other details for this game. Thanks in advance!

Comment (RadWolfie) [reply src/CxbxKrnl/HLEDataBase.cpp]: XACTENLT library is actually LTCG version of XActEng library. We cannot support it. May I advice you to change the description to...
`(Unluckily, it use LTCG version of the library.)`

**Comment by RadWolfie** [comment]
@jarupxx It is safe to say it works as intended? I do not have any titles with XActEng to verify against at the moment.

File: src/CxbxKrnl/HLEDataBase.cpp (hunk 2 above)
Comment (PatrickvL) [reply src/CxbxKrnl/HLEDataBase.cpp]: @jarupxx Thanks for creating a compatibility report

**Comment by jarupxx** [comment]
@RadWolfie I finally test this XactEng database v2. It is safe as symbols are detected correctly by database v2.

**Comment by RadWolfie** [comment]
Thanks for verifying with XActEng database v2. Merging now.

**Pull request merged**